package.json

```diff
@@ -14,7 +14,7 @@
     "@astrojs/netlify": "^5.2.1",
     "@astrojs/tailwind": "^5.1.0",
     "@auth/core": "^0.18.6",
-    "@sentry/astro": "^8.7.0",
+    "@sentry/astro": "^8.8.0",
```

Comment:
bumped to 8.8.0 because it resolves an ESM import issue I experienced on first startup